{{cookiecutter.project_name}}/README.md

```diff
@@ -3,24 +3,28 @@
 
 {{cookiecutter.app_description_summary}}
 
-![{{cookiecutter.project_name}} by {{cookiecutter.org_name}}](Applications/App/Resources/Images/LogoFull.png?raw=true)
+![{{cookiecutter.project_name}} by {{cookiecutter.org_name}}](Applications/{{cookiecutter.app_name}}App/Resources/Images/LogoFull.png?raw=true)
 
-Prerequisites
+Windows Build Instructions
+--------------------------
+
+Instructions for the Windows build are given here.  The app can also be built on Linux and macOS. See: [3D Slicer Developer Wiki](http://wiki.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers)
```

Comment:
`Instructions for the Windows build are given here` -> `Instructions for the Windows build are listed below`

What do you think of moving the build instructions into a `BUILD.md` document ?

Reply:
Sure we can do that.  Should we then include all 3 platforms?

Reply:
> Should we then include all 3 platforms?

That would make sense

Reply:
I will let you decide if you want to include instructions for the other two platforms now or at a later time.

Reply:
For reference, here is what I did for CellLocator. See https://github.com/BICCN/cell-locator/blob/master/BUILD.md